core/src/migrations/000055-errorLevelForExports.ts

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  up: async function (migration, DataTypes) {
+    await migration.addColumn("exports", "errorLevel", {
+      type: DataTypes.STRING(191),
+      allowNull: true,
+      defaultValue: null,
+    });
+
```

Comment:
From chat: Maybe we add another column to keep track of the Export really being "done" - the retries are completed